kubecost-cloud-savings.md

```diff
@@ -0,0 +1,13 @@
+# Kubecost Cloud: Savings
+
+{% hint style="info" %}
+This documentation should only be consulted when using Kubecost Cloud! For information about the Savings page for self-hosted Kubecost, see [here](savings.md).
+{% endhint %}
+
+The Savings page provides miscellaneous functionality to help you use resources more effectively and assess wasteful spending. In the center of the page, you will see your estimated monthly savings available.
+
+## Savings insights
```

Comment:
New line after headings and before lists of elements like the bulleted list below.